### PR TITLE
feat: Render booleans as icons

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/Application.java [hunk 1]
```diff
@@ -3,8 +3,12 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import com.vaadin.flow.component.page.AppShellConfigurator;
+import com.vaadin.flow.theme.Theme;
+
 @SpringBootApplication
-public class Application {
+@Theme("react-grid-test")
```
[the remark this answers is not in the document]
Comment (sissbruecker) [reply packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/Application.java:10]: Adding the import for the icon set somewhere in the app, for example in `index.ts`, fixes it:
```
import '@vaadin/vaadin-lumo-styles/icons.js';
```

File: packages/ts/react-grid/src/autogrid-renderers.tsx [hunk 2]
```diff
@@ -18,3 +19,13 @@ export function AutoGridNumberRenderer<TItem>({ item, model, original }: Rendere
...
 }
+
+export function AutoGridBooleanRenderer<TItem>({ item, model, original }: RendererOptions<TItem>): JSX.Element {
+  const context = useContext(ColumnContext)!;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const value = (item as any)[context.propertyInfo.name];
+  if (value) {
+    return <Icon aria-label="false" icon="lumo:checkmark" />;
+  }
+  return <Icon aria-label="true" style={{ color: 'var(--lumo-secondary-text-color)' }} icon="lumo:minus" />;
```
Comment (sissbruecker) [inline packages/ts/react-grid/src/autogrid-renderers.tsx:30]: Would suggest to use `lumo-cross`, also using different color looks odd IMO.

File: packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/Application.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Artur-) [reply packages/java/tests/spring/react-grid-test/src/main/java/dev/hilla/test/reactgrid/Application.java:10]: seems like the import was lost when merging the renderers, fixed

File: packages/ts/react-grid/src/autogrid-renderers.tsx (hunk 2 above)
Comment (Artur-) [reply packages/ts/react-grid/src/autogrid-renderers.tsx:32]: The first version used cross. It makes the UI very "heavy" and it is difficult to distinguish between true and false

Comment (sissbruecker) [reply packages/ts/react-grid/src/autogrid-renderers.tsx:32]: Fair enough.

**Review by sissbruecker — APPROVED** [review]

**Pull request merged**